### PR TITLE
fix: docs

### PR DESCRIPTION
**Pull request by storopoli**
- add CompatHelper for `test/` and `docs/` subdirs `Project.toml`s

### CODE REVIEW

File: .github/workflows/CompatHelper.yml [hunk 1]
```diff
@@ -6,11 +6,14 @@ on:
...
+        subdir: ["", "docs", "test"]
     steps:
       - name: Pkg.add("CompatHelper")
         run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
       - name: CompatHelper.main()
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+        run: julia -e 'using CompatHelper; CompatHelper.main(; subdirs=["${{ matrix.subdir }}"])'
```
Comment (rikhuijzer) [inline .github/workflows/CompatHelper.yml:19]: For subdirs maybe set to only keep new and remove old compat? Test and docs don't have to be so flexible in terms of their compat entries

File: test/Project.toml [hunk 2]
```diff
@@ -12,7 +12,7 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 
 [compat]
-CSV = "0.9, 1"
+CSV = "0.10, 1"
```
Comment (rikhuijzer) [inline test/Project.toml:15]:
```suggestion
CSV = "1"
```

File: docs/Project.toml [hunk 3]
```diff
@@ -9,9 +9,9 @@ PlutoStaticHTML = "359b1769-a58e-495b-9770-312e911026ad"
 TuringGLM = "0004c1f4-53c5-4d43-a221-a1dac6cf6b74"
 
 [compat]
-CSV = "0.9, 1"
-CairoMakie = "0.7"
+CSV = "0.10, 1"
```
Comment (rikhuijzer) [inline docs/Project.toml:12]:
```suggestion
CSV = "1"
```

File: .github/workflows/CompatHelper.yml (hunk 1 above)
Comment (storopoli) [reply .github/workflows/CompatHelper.yml]: Like in a802653 ?

Comment (storopoli) [reply .github/workflows/CompatHelper.yml]: Seems that this breaks CI. Any suggestions?

**Comment by storopoli** [comment]
@rikhuijzer let's deal with dropping compats in the CompatHelper PRs?

**Review by rikhuijzer — APPROVED** [review]

**Pull request merged**